### PR TITLE
fix: correct withdrawal prompt wait time

### PR DESCRIPTION
**Pull request by zhangyouxin**
fix: correct withdrawal prompt wait time

### CODE REVIEW

File: src/utils/dateUtils.ts [hunk 1]
```diff
@@ -1 +1,22 @@
...
+  if (waitSeconds < secondsPerHour) {
+    return `${Math.round(waitSeconds / secondsPerMinute)} minutes`;
+  } else if (waitSeconds >= secondsPerHour && waitSeconds < secondsPerday) {
+    return `${Math.round(waitSeconds / secondsPerHour)} hours`;
+  } else if (waitSeconds >= secondsPerday) {
+    return `${Math.round(waitSeconds / secondsPerday)} days`;
+  } else {
+    return "unkonwn time";
+  }
+};
```
Comment (homura) [inline src/utils/dateUtils.ts:22]: we should add unit tests for utility functions, or maybe we can replace them with [date-fns#intervalToDuration](https://date-fns.org/v2.28.0/docs/intervalToDuration)

File: src/light-godwoken/lightGodwoken.ts [hunk 2]
```diff
@@ -65,14 +65,17 @@ export default abstract class DefaultLightGodwoken implements LightGodwokenBase
...
   abstract getErc20Balances(payload: GetErc20Balances): Promise<GetErc20BalancesResult>;
-  abstract getBlockProduceTime(): number | Promise<number>;
   abstract getWithdrawal(txHash: Hash): Promise<unknown>;
   abstract getBuiltinErc20List(): ProxyERC20[];
   abstract getBuiltinSUDTList(): SUDT[];
   // abstract listWithdraw(): Promise<WithdrawResultWithCell[]>;
   abstract getVersion(): GodwokenVersion;
   abstract withdrawWithEvent(payload: WithdrawalEventEmitterPayload): WithdrawalEventEmitter;
 
+  getBlockProduceTime(): number {
```
Comment (homura) [inline src/light-godwoken/lightGodwoken.ts:75]:
```suggestion
  // in milliseconds
  getBlockProduceTime(): number {
```

File: src/light-godwoken/constants/lightGodwokenConfig.ts [hunk 3]
```diff
@@ -145,104 +146,111 @@ const v0ConfigLina: LightGodwokenConfig = {
...
-          script_type_hash: "0x50704b84ecb4c4b12b43c7acb260ddd69171c21b4c0ba15f3c469b7d143f6f18",
-          cell_dep: {
-            out_point: {
-              tx_hash: "0x9caeec735f3cd2a60b9d12be59bb161f7c61ddab1ac22c4383a94c33ba6404a2",
-              index: "0x0",
-            },
-            dep_type: "code",
+const v1ConfigAggron: LightGodwokenConfig = {
+  layer1Config: layer1ConfigAggron,
+  layer2Config: {
```
Comment (homura) [inline src/light-godwoken/constants/lightGodwokenConfig.ts:156]: please add a permalink to link to godwoken-info

File: src/utils/dateUtils.ts (hunk 1 above)
Comment (zhangyouxin) [reply src/utils/dateUtils.ts:22]: add test cases

File: src/light-godwoken/lightGodwoken.ts (hunk 2 above)
Comment (zhangyouxin) [reply src/light-godwoken/lightGodwoken.ts:76]: added comments

File: src/light-godwoken/constants/lightGodwokenConfig.ts (hunk 3 above)
Comment (zhangyouxin) [reply src/light-godwoken/constants/lightGodwokenConfig.ts:159]: added permalink

**Review by homura — APPROVED** [review]

**Pull request merged**